internal/gouging/gouging.go

```diff
@@ -158,7 +158,7 @@ func checkPriceGougingHS(gs api.GougingSettings, hs *rhpv2.HostSettings) error {
 // TODO: if we ever stop assuming that certain prices in the pricetable are
 // always set to 1H we should account for those fields in
 // `hostPeriodCostForScore` as well.
-func checkPriceGougingPT(gs api.GougingSettings, cs api.ConsensusState, txnFee types.Currency, pt *rhpv3.HostPriceTable) error {
+func checkPriceGougingPT(gs api.GougingSettings, cs api.ConsensusState, _ types.Currency, pt *rhpv3.HostPriceTable) error {
```

Comment:
I had to add this to make `lint` pass. I didn't want to remove it all the way because it touches upon too many files, this parameter is removed in `dev` so perhaps we can let it slide here and keep the hotfix LoC changes to a minimum.